poppy/poppy_core.py

```diff
@@ -1162,37 +1164,31 @@ def propagate_mono(self, wavelength=2e-6, normalize='first', save_intermediates=
             * 'last' = set total flux = 1 after the entire optical system.
             * 'first=2' = set total flux = 2 after the first optic (used for debugging only)
         display_intermediates : bool
-            Should intermediate steps in the calculation be displayed on screen? Default False
-        save_intermediates : bool
-            Should intermediate steps in the calculation be saved to disk? Default False.
-            If this is True, then setting `poly_weight` controls whether intermediate optical planes are actually saved to *disk* by this routine
-            (for the monochromatic case) or are passed back up via memory and handled in `calcPSF` (for the polychromatic case).
-        poly_weight : float
-            is this being called as part of a polychromatic calculation?
-            if not, set this to None. if so, set this to the weight for
-            that wavelength. (This is used only for properly normalizing the
-            multiwavelength FITs file written to disk if save_intermediates is set.)
-
+            Should intermediate steps in the calculation be displayed on screen? Default: False.
+        retain_intermediates : bool
+            Should intermediate steps in the calculation be retained? Default: False.
+            If True, the second return value of the method will be a list of `poppy.Wavefront` objects
+            representing intermediate optical planes from the calculation.
 
+        Returns
+        -------
+        final_wf : fits.HDUList
+            The final result of the monochromatic propagation as a FITS HDUList
+        intermediate_wfs : list
+            A list of `poppy.Wavefront` objects representing the wavefront at intermediate optical planes.
+            The 0th item is "before first optical plane", 1st is "after first plane and before second plane", and so on.
+            (n.b. This will be empty if `retain_intermediates` is False.)
         """
 
         if conf.enable_speed_tests:
             t_start = time.time()
-        if self.verbose: _log.info(" Propagating wavelength = %g meters %s" % (wavelength, "" if poly_weight is None else " with weight=%.2f" % poly_weight))
+        if self.verbose: _log.info(" Propagating wavelength = %g meters %s")
```

Comment:
This format statement appears to lack any formatting function call to expand the `%g` and `%s`?